### ISSUE TITLE
Unable to get info of person associated with the account

### ISSUE THREAD

**Issue by sarabveer** [costastf/locationsharinglib#44]
When using version 3.0.6, running `get_authenticated_person()` returns `None`.

test.py
```python
from locationsharinglib import Service
service = Service(username, password, cookies_file)
print(service.get_authenticated_person())
```

Output:
```
None
```

Due to this, the google account used also does not come up in `get_all_people()`.

This works in 2.0.11

**Comment by sarabveer** [comment]
So I figured out the problem,

https://github.com/costastf/locationsharinglib/blob/master/locationsharinglib/locationsharinglib.py#L109

Here, it throws an error as there is no battery data for the authenticated person.

Replacing it with this fixed the issue:
```
            try:
                self._charging = data[13][0]
            except:
                self._charging = None
            try:
                self._battery_level = data[13][1]
            except:
                self._charging = None
```

**Comment by ronmichel** [comment]
I fixed it like this:   

```
 def get_authenticated_person(self):
        """Retrieves the person associated with this account"""
        
        try:
            output = self._get_data()
            person = Person([
                self.email,
                output[9][1],
                None,
                None,
                None,
                None,
                [
                    self.email,
                    None,
                    self.email,
                    self.email
                ],
                None,
                None,
                None,
                None,
                None,
                None,
                [
                    None,
                    None
                ]
            ])
        except (IndexError, TypeError, InvalidData):
            self._logger.debug('Missing essential info, cannot instantiate authenticated person')
            return None
        return person
```

**Comment by sarabveer** [comment]
Fixed with 3.0.8
